Formula/maxbin2.rb

```diff
@@ -20,18 +20,22 @@ class Maxbin2 < Formula
   depends_on "hmmer"
   depends_on "idba"
   depends_on "perl" unless OS.mac?
+  depends_on "openssl"
 
   def install
     ENV.prepend_path "PERL5LIB", Formula["bioperl"].libexec/"lib/perl5"
     ENV.prepend_create_path "PERL5LIB", prefix/"perl5/lib/perl5"
     system "cpanm", "--self-contained", "-l", prefix/"perl5", "IO::Socket::SSL", "LWP::Simple"
-    cd "src" do
-      system "make"
-    end
+    system "make", "-C", "src"
     (libexec/"src").install "src/MaxBin"
-    rm_r "src"
```

Comment:
This line is needed to prevent the following line from installing the `src` directory in `libexec`.